### DISCUSSION TITLE
v0.20.0-alpha.1 - Prototype Performance Data release

### DISCUSSION THREAD

**Discussion by atc0005** [atc0005/check-vmware#322] [Dev/Release Candidate]
**NOTE**: This is not considered a stable release.  

While the plugins are likely safe to use, initial prototype support for emitting performance data has been added for the `check_vmware_datastore` plugin. 

References:

- GH-315
- GH-321
- atc0005/go-nagios#81
- atc0005/go-nagios@1db1acf
- the (temporary) `i321-prototype-datastore-perfdata-support` branch
- commit 356565fd2ec3b59e76b635207acbe9e328badbcd

<hr /><em>This discussion was created from the release <a href='https://github.com/atc0005/check-vmware/releases/tag/v0.20.0-alpha.1'>v0.20.0-alpha.1 - Prototype Performance Data release</a>.</em>

**Comment by atc0005** [comment]
@byolock Ready for your testing.

**Reply by Byolock** [reply]
Thank you, I'm on vacation this week but will try to get back to you with first results on the next week.

**Reply by atc0005** [reply]
Thanks, and enjoy the vacation!

**Reply by atc0005** [reply]
Follow-up: 

- v0.19.1 is a dependency update only (doesn't include the PerfData output we're testing in v0.20.0-alpha.x releases)
- [v0.20.0-alpha.3](https://github.com/atc0005/check-vmware/releases/tag/v0.20.0-alpha.3) is the current version available for testing
